annif/backend/http.py

```diff
@@ -11,6 +11,33 @@
 class HTTPBackend(backend.AnnifBackend):
     name = "http"
 
+    @property
+    def is_trained(self):
+        return self._get_project_info('is_trained')
+
+    @property
+    def modification_time(self):
+        return self._get_project_info('modification_time')
+
+    def _get_project_info(self, key):
+        params = self._get_backend_params(None)
+        try:
+            req = requests.get(params['endpoint'].replace('/suggest', ''))
```

Comment:
Having to do this URL modification here is probably an indication that the `endpoint` parameter should be pointing at a root resource of the API (either the root of the API, or the project level), instead of directly at the `/suggest` method. That would be a backwards incompatible change though. Maybe that could be fixed in a separate PR, since this one is already quite complex.